### DISCUSSION TITLE
Need way to close epoptes cleanly

### DISCUSSION THREAD

**Discussion by sutula** [epoptes/epoptes#32] [Q&A]
Current epoptes does not seem to cleanly shutdown.

There is a "Quit" in the File menu.  When using this, I find that two processes apparently spawned by epoptes are still running:

    dbus-launch --autolaunch=...
    /usr/bin/dbus-daemon --fork ...

I can get further details if necessary, but hope that the above is sufficient.  I'd recommend that epoptes remember process IDs of processes that are created, and ensure that these are killed during shutdown.  Without this fix, processes are left running on the server.

**Comment by alkisg** [comment]
If dbus-daemon wasn't already running for you before starting epoptes, there's probably something weird in the way you launch it?

Are you sure those processes are launched by epoptes? Can you paste the output of `ps faux`?

**Comment by alkisg** [comment]
Closing because of no feedback.
